src/group_impl.h

```diff
@@ -11,49 +11,125 @@
 #include "field.h"
 #include "group.h"
 
-/* These points can be generated in sage as follows:
+/* These exhaustive group test orders and generators are chosen such that:
+ * - The field size is equal to that of secp256k1, so field code is the same.
+ * - The curve equation is of the form y^2=x^3+B for some constant B.
+ * - The order is less than 1000 to permit exhaustive testing.
+ * - The order is a multiple of 3 plus 1, enabling the endomorphism optimization.
+ * - The generator is 2*P, where P has X coefficient equal to 1.
  *
- * 0. Setup a worksheet with the following parameters.
- *   b = 4  # whatever CURVE_B will be set to
- *   F = FiniteField (0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEFFFFFC2F)
- *   C = EllipticCurve ([F (0), F (b)])
+ * They can generated with the following Sage code:
```

Comment:
We have a sage directory, perhaps best to move this over? My first attempt at copy & pasting this messed up indentation.

Reply:
Yeah, done.